airbyte-integrations/connectors/source-intercom/source_intercom/spec.json

```diff
@@ -4,20 +4,85 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Source Intercom Spec",
     "type": "object",
-    "required": ["access_token", "start_date"],
-    "additionalProperties": false,
+    "required": ["start_date"],
+    "additionalProperties": true,
```

Comment:
@sherifnada I have updated python and java part, but I will need to check this update with java dev

Reply:
the spec looks right! for the record, facebook marketing is a good example of this pattern @midavadim

Reply:
@sherifnada I have made a few fixes and tested in UI part (actually with curl - because redirect url is airbyte.io). Facebook example helped a lot! Thank you!